frontend/pages/hosts/details/DeviceUserPage/DeviceUserPage.tsx

```diff
@@ -353,14 +374,15 @@ const DeviceUserPage = ({
               mdmEnabledAndConfigured={
                 !!globalConfig?.mdm.enabled_and_configured
               }
-              mdmConnectedToFleet={!!host.mdm.connected_to_fleet}
-              diskEncryptionStatus={
+              connectedToFleetMdm={!!host.mdm.connected_to_fleet}
```

Comment:
Need to verify that this doesn't show up for Linux hosts as there isn't a concept of Linux MDM the same way there is for Mac/Win.

Reply:
This should be testable now with a Linux host as nothing we're doing on the backend changes the fact that there's no Linux MDM.

Reply:
This prop is only used in conjunction with (a)` macDiskEncryptionStatus === "action_required"` to determine whether to (b) `showMacDiskEncryptionKeyResetRequired` – since (a) should only be possible for `darwin` hosts, there shouldn't be any issue here.

[clarifying rename](https://github.com/fleetdm/fleet/pull/23714/commits/f91266a87e6d749de44b534cab9297ecf05bd87c) that should make those relationships more clear.

Does that make sense or am I missing something?

Reply:
Will, as noted, test all of this today